Error when calling super()

After updating my starter kit to match, I am getting this error in production when running npm run start. It seems to be caused by my React class calling the super constructor. The base starter kit doesn't seem to call super yet, but you will definitely run into this error after adding a few React components on top of the base. 

```
return obj && typeof _symbol2.default !== "undefined" && obj.constructor === _symbol2.default ? "symbol" : typeof obj === "undefined" ? "undefined" : (0, _typeof3.default)(obj);
```

Seemed to have tracked it here: https://phabricator.babeljs.io/T6670 and here: https://phabricator.babeljs.io/T6644. The cause seems to be updating the babel-runtime.

@patrickheeney As soon as the make starts, on every file. It finally says the build it complete and then it errors out with:

```
Cannot find module '/mypath/node_modules/babel-patch/packages/babel-polyfill/lib/index.js'
```

--- a/package.json
+++ b/package.json
@@ -38,9 +38,9 @@
     "yargs": "^3.18.0"
   },
   "devDependencies": {
-    "babel-core": "^6.3.15",
-    "babel-eslint": "^5.0.0-beta4",
-    "babel-loader": "^6.1.0",
+    "babel-core": "^6.3.17",
+    "babel-eslint": "^5.0.0-beta6",
+    "babel-loader": "^6.2.0",
     "babel-plugin-add-module-exports": "^0.1.1",
     "babel-plugin-react-transform": "^2.0.0-beta1",
     "babel-plugin-transform-runtime": "^6.3.13",
@@ -48,7 +48,7 @@
     "babel-preset-react": "^6.3.13",
     "babel-preset-stage-0": "^6.3.13",
     "babel-register": "^6.3.13",
-    "babel-runtime": "^6.3.13",
+    "babel-runtime": "^6.3.19",
     "chai": "^3.4.1",
     "chai-as-promised": "^5.1.0",
     "connect-history-api-fallback": "^1.1.0",

